applications/pelion_client/src/modules/pelion_trace.c

```diff
@@ -10,6 +10,14 @@
 #define MODULE pelion_trace
 #include <caf/events/module_state_event.h>
 
+#include <logging/log.h>
+LOG_MODULE_REGISTER(MODULE, CONFIG_PELION_CLIENT_PELION_LOG_LEVEL);
+
+
+static void trace_printer(const char *str)
+{
+	LOG_INF("%s", log_strdup(str));
```

Comment:
@nordic-krch if only we used different, unused formatting symbol the compatibility of printk with other printing functions could have been maintained.
Now it's probably much too late.